app/helpers/application_helper.rb

```diff
@@ -1,4 +1,8 @@
 module ApplicationHelper
+  def availability_collection
+    [:flexible, :morning, :afternoon, :evening, :workday, :weekend]
```

Comment:
The concern in a controller concern, used by the controllers and this is a helper method to be used by the views. I am not sure if concerns are supposed to be used somehow by the views.